`__all__` is erroneously flagged with W8301

**Describe the bug**

When `pylint`ing my code `perflint` erroneously flags `__all__` with W8301.

**To reproduce**

`perflint.py`
```python
"""perflint test module."""

__all__ = ["dummy"]

def dummy() -> None:
    """Dummy docstring."""
    raise NotImplementedError

```

Run `pylint`:

```sh
$ pylint --load-plugin=perflint perflint.py
************* Module perflint
perflint.py:3:10: W8301: Use tuple instead of list for a non-mutated sequence (use-tuple-over-list)

------------------------------------------------------------------
Your code has been rated at 6.67/10 (previous run: 3.33/10, +3.33)
```

**Expected behavior**

`__all__` should not be flagged.

While [`__all__` [...] must be a sequence of strings](https://docs.python.org/3/reference/simple_stmts.html?highlight=__all__#the-import-statement), and a tuple is a sequence, the standard is to declare `__all__` as `list[str]`.

--- a/perflint/for_loop_checker.py
+++ b/perflint/for_loop_checker.py
@@ -3,6 +3,8 @@ from astroid import nodes
 from astroid.helpers import safe_infer
 from pylint.checkers import BaseChecker
 from pylint.checkers import utils as checker_utils
+from pylint.interfaces import INFERENCE
+
 
 iterable_types = (
     nodes.Tuple,
@@ -141,7 +143,7 @@ class LoopInvariantChecker(BaseChecker):
             "Global name lookups in Python are slower than local names.",
         ),
         "R8203": (
-            "Try..except blocks have an overhead. Avoid using them inside a loop unless you're using them for control-flow.",
+            "Try..except blocks have an overhead. Avoid using them inside a loop unless you're using them for control-flow. Rule only applies to Python < 3.11.",
             "loop-try-except-usage",
             "Avoid using try..except within a loop.",
         ),
@@ -336,7 +338,7 @@ class LoopInvariantChecker(BaseChecker):
     @checker_utils.only_required_for_messages("loop-try-except-usage")
     def visit_tryexcept(self, node: nodes.Try) -> None:
         if self._loop_level > 0:
-            self.add_message("loop-try-except-usage", node=node)
+            self.add_message("loop-try-except-usage", node=node, confidence=INFERENCE)
 
     @checker_utils.only_required_for_messages("memoryview-over-bytes")
     def visit_subscript(self, node: nodes.Subscript) -> None:

--- a/perflint/list_checker.py
+++ b/perflint/list_checker.py
@@ -29,6 +29,8 @@ class ListChecker(BaseChecker):
         if len(node.targets) > 1:
             return
         if isinstance(node.targets[0], nodes.AssignName):
+            if node.targets[0].name == "__all__":
+                return
             self._lists_to_watch[-1][node.targets[0].name] = node.targets[0]
 
     def visit_module(self, node: nodes.Module):

--- a/tests/test_list_checker.py
+++ b/tests/test_list_checker.py
@@ -16,6 +16,15 @@ class TestListMutationChecker(BaseCheckerTestCase):
         with self.assertAddedMessage("use-tuple-over-list"):
             self.walk(test_func)
 
+    def test_const_all_be_ignored(self):
+        test_func = astroid.extract_node("""
+        def test(): #@
+            __all__ = [1,2,3,4]
+        """)
+
+        with self.assertNoMessages():
+            self.walk(test_func)
+
     def test_mutated_list_by_method(self):
         test_func = astroid.extract_node("""
         def test(): #@

